environments/prod/prod.tfvars

```diff
@@ -176,6 +176,13 @@ frontends = [
       },
     ]
   },
+  {
+    name             = "sscs-cor-apex"
```

Comment:
this will just get you a 404, as there's no ingress record for it or handling, is that what you're hoping to achieve?

Reply:
This is the fix for the SSL errror for the problem record yes. Any work to make sure non www. domains are forwarded correctly should also be done but I believe that's a seperate problem already for most of the services.

Reply:
The other option is to delete the apex record so it responds with a Host Not Found NS error like the other services currently do.

Reply:
having it consistent with the other services would be best IMO.